[Comms handbook] Add ready-for-review deadline reminder template (#2754)
kubernetes/sig-release: release-team/role-handbooks/communications/templates/feature-blog-messages.md

  - `<<BLOG_READY>> WEEKDAY, DAY MONTH`: Deadline for the Feature Blogs being ready for review, which is usually on week 13
- `LINK`: Add a link to the release document, such as `https://github.com/kubernetes/sig-release/tree/master/releases/release-1.33#timeline`

# Feature Blog PR

## "ready for review" deadline reminder
This is a message to send Feature Blog authors in their Feature Blog PR if they haven't marked it as "ready for review" days before the deadline.
```

> Hi @xx 👋 -- this is YOURNAME (@your_github_id) from the x.yy Communications Team!

Reviewer: While I'm at this, I'd just suggest updating the ones we can quickly fix

```suggestion
Hi @xx 👋 -- this is YOURNAME (@your_github_id) from the vX.yy Communications Team!
```

> Just a friendly reminder that we are approaching the feature blog "ready for review" deadline: **<<BLOG_READY>> WEEKDAY, DAY MONTH**. By this deadline, your blog should be complete with all content finalized for review. If you are still working on it, please prioritize finishing it by the deadline mentioned above.

Reviewer: Reading this once again, I think I'd suggest phrasing a bit differently, mainly for two reasons:

- Blog is optional, and it is very possible that the KEP owners get carried away with other priorities, and they cannot complete in time
- Blog write-up schedule usually is very close to Docs deadline; Docs is a mandatory item, and they should surely prioritise that before blogs

So, what about rewording to something like?

```suggestion
Just a friendly reminder that we are approaching the feature blog "ready for review" deadline: **<<BLOG_READY>> WEEKDAY, DAY MONTH**. We ask you to have the blog in non-draft state, and most of the write-up to be complete, so that we can start the blog review from SIG Docs Blog team.
```

If you have any questions or need help, please don't hesitate to reach out to me or any of the Communications Team members. We are here to help you!
```
In this template, you'd need to update the following placeholders:
- `@xx`: Mention Feature Blog PR author(s)
- `YOURNAME`: Your name, as you prefer
- `your_github_id`: Your GitHub ID

> - `x.yy`: The upcoming Kubernetes version, such as 1.33, 1.34, etc.

Reviewer: While I'm at this, I'd just suggest updating the ones we can quickly fix

```suggestion
- `vX.yy`: The upcoming Kubernetes version, such as v1.33, v1.34, etc.
```
